plugins/chat/db/migrate/20241003122030_add_notification_level_to_user_chat_channel_memberships.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+class AddNotificationLevelToUserChatChannelMemberships < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def up
+    execute <<~SQL
+      DROP INDEX CONCURRENTLY IF EXISTS user_chat_channel_memberships_index
+    SQL
+
+    add_column :user_chat_channel_memberships, :notification_level, :integer, null: true
```

Comment:
@dbattersby @markvanlan @nattsw We should not be adding a column or [changing a column's defaults](https://github.com/discourse/discourse/pull/29080/files#diff-d4b35a813304e060b02a2a2b55c5aadca4d2034ad9d97838d35d3d380a8673a5R21-R22) in a migration where `disable_ddl_transaction!` is used. Migrations with transactions disabled need to be idempotent so that we can run it again even if the migration fails.